Zinc/Private/ZincRepoIndex.m

```diff
@@ -226,6 +226,12 @@ - (void) removeBundle:(NSURL*)bundleResource
     }
 }
 
+- (void) purgeBundleWithID:(NSString *)bundleID {
```

Comment:
there'a already a method called `delete` which does something weaker than this, so I named this `purge`

`delete` does not actually delete the bundle entry in the index. It keeps the bundle entry, and simply deletes some properties in that entry.